src/components/Radio/RadioItem/RadioItem.module.css

```diff
@@ -49,6 +50,7 @@
   flex-direction: row;
   align-items: center;
   justify-content: flex-start;
+  font-family: var(--font-family);
```

Comment:
I see the `font-family` prop is repeated in many files.

Lets add this to the css-reset.
```css
* {
  ...
  font-family: var(--font-family);
}
```